goodtablesio/tasks.py

```diff
@@ -1,32 +1,74 @@
-import os
-
+import dataset
```

Comment:
I've run into it sharing the connection - https://github.com/celery/celery/issues/1564 (may be we could use threads for celery). Just this optimization question not for now I suppose.